server/src/frameworks/Ape/ApeIndexer.ts

```diff
@@ -0,0 +1,22 @@
+import path from "path";
+import { WorkspaceFolder } from "vscode-languageserver-protocol";
+import { decodeUriAndRemoveFilePrefix } from "../../utils";
+import { ProjectIndexer } from "../base/ProjectIndexer";
+import { ApeProject } from "./ApeProject";
+
+export class ApeIndexer extends ProjectIndexer {
+  public async index(folder: WorkspaceFolder) {
+    const uri = decodeUriAndRemoveFilePrefix(folder.uri);
+    const configFiles = await this.fileRetriever.findFiles(
```

Comment:
It could happen that a dependency is an ape project on itself (has an ape config file), and we want to avoid indexing those as projects. It's the same approach we currently follow with all framework adapters.